enclave_manager/Dockerfile

```diff
@@ -232,8 +230,6 @@ RUN echo "Building avalon enclave manager" \
 # Build Final image and install dependent modules
 FROM base_image as final_image
 
-ENV http_proxy=$http_proxy
-ENV https_proxy=$https_proxy
```

Comment:
Have we tested SGX h/w with this change. Please verify if connection to IAS server works after removing this.